doc/radosgw/account.rst

```diff
@@ -0,0 +1,129 @@
+===============
+ User Accounts
+===============
+
+.. versionadded:: Squid
+
+Ceph Object Gateway supports user accounts as an optional feature to enable
```

Comment:
The Ceph Object ...?  not sure what usage we want to converge on.

Maybe italicize _user accounts_?

Is this congruent with AWS S3, or an RGW-specific feature?